inc/config.php

```diff
@@ -49,28 +49,76 @@
 
 Config::detectRootDoc();
 
-if (!file_exists(GLPI_CONFIG_DIR . "/config_db.php")) {
+$conf_exists = file_exists(GLPI_CONFIG_DIR . "/db.yaml");
+if (!$conf_exists && file_exists(GLPI_CONFIG_DIR . "/config_db.php")) {
+    //convert old config file to new one
+    $oldconf = file_get_contents(GLPI_CONFIG_DIR . "/config_db.php");
+    $matches = [];
+
+    preg_match('/dbhost\s*=\s*["\'](.+)["\'];/', $oldconf, $matches);
```

Comment:
This could fail if the file contains a commented line with old db host, as it may use the wrong value
The most secure way to get values from the  config file, without including it directly, would be to use php tokenizer. Do you want me to try to do the piece of code using it to see of it is a good solution ?

Reply:
lt's go :)